server/filestore.go

```diff
@@ -2289,6 +2289,12 @@ func (mb *msgBlock) readIndexInfo() error {
 	mb.last.ts = readTimeStamp()
 	dmapLen := readCount()
 
+	// Check if this is a short write index file.
```

Comment:
I am not too familiar with the storage code - if we just nuke the index will it be rebuilt? Whats the impact of that?

Reply:
Yes will rebuild from raw data.

Reply:
I want to understand a bit more why she got the short write, I think there may be a timing bug somewhere as well.

Reply:
Could also just be a kill -9 at the wrong time? I think there was a lot of hard resetting the server because the other bug is that she caused it to become unresponsive (which I now cant reproduce)